tensorflow-ops/src/TensorFlow/Minimize.hs

```diff
@@ -22,11 +24,15 @@ module TensorFlow.Minimize
     ( Minimizer
     , minimizeWith
     , gradientDescent
+    , OneOfAdamDataTypes
     , AdamConfig(..)
     , adam
     , adam'
     ) where
 
+import Data.Complex
```

Comment:
Could you enumerate the imported symbols? We've been doing this quite consistently.

Reply:
Done.